### ISSUE TITLE
Enable filter to only show in-stock products and SKUs

### ISSUE THREAD

**Issue by nbezalwar** [vendure-ecommerce/vendure#870]
I believe this will need update to both shop-api and admin-api.

**Comment by pravin-logicap** [comment]
@michaelbromley: 
filter can apply on native DB fields and we can filter that out with typeORM query builder. 
But in variants we derived stockLevel from result.
So if we want to apply filter based on stockLevel , Can we filter it out in resolver ? (eg once we get all items) or is there other way to do? 

I am able to get the field stockLevel  at product level based stockLevel  result from variants.

**Comment by michaelbromley** [comment]
@pravin-logicap does the above implementation for Elastic work for your needs?

**Comment by pravin-logicap** [comment]
Yes it worked. But we have change in elasticsearch plugin code as mentioned above. (point 1 and point 2)

**Comment by Izayda** [comment]
I can get this
